src/builder.rs

```diff
@@ -0,0 +1,95 @@
+use std::path::{Path, PathBuf};
+
+use crate::Search;
+
+/// Builder for a [`Search`] instance, allowing for more complex searches
+pub struct SearchBuilder {
+    /// The location to search in, defaults to the current directory
+    search_location: PathBuf,
+    /// The search input, defaults to any word
```

Comment:
/// The search input, defaults to search for every word.

This would be better ig